docs/en/04_Changelogs/4.10.0.md

```diff
@@ -0,0 +1,35 @@
+# 4.10.0 (unreleased)
+
+## Overview
+
+- [Regression test and Security audit](#audit)
+- [Dropping support for PHP 7.1 and PHP 7.2](#phpeol)
+- [Features and enhancements](#features-and-enhancements)
+- [Bugfixes](#bugfixes)
+
+
+## Regression test and Security audit{#audit}
+
+This release has been comprehensively regression tested and passed to a third party for a security-focused audit. 
+
+While it is still advised that you perform your own due diligence when upgrading your project, this work is performed to ensure a safe and secure upgrade with each recipe release.
+
+## Dropping support for PHP 7.1 and PHP 7.2{#phpeol}
+
+We've recently updated our [PHP support policy](/Getting_Started/Server_Requirements#php). The immediate affects of this changes are:
+
+- The Silverstripe CMS Recipe release 4.10.0 drops support for PHP 7.1 and PHP 7.2. Those two PHP releases have been end-of-life for several years now and continued support would detract effort from more valuable work.
+- The 4.11 minor release will drop support for PHP 7.3 later this year.
+- We expect to drop support for PHP 7 altogether around January 2023.
+
```

Comment:
We have a blog post which we can link to here, but we don't have a URL for it yet.